srcpkgs/python-pyzbar/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'python-pyzbar'
+pkgname=python-pyzbar
+version=0.1.7
+revision=2
+noarch=yes
+wrksrc="pyzbar-${version}"
+build_style=python-module
+pycompile_module="pyzbar"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="libzbar python-enum34 python-dateutil"
```

Comment:
- `dateutil` is not required
- `depends="libzbar python-enum34 python-pathlib"`

Note: `libzbargtk` and `libzbarqt` should be splitted out of `libzbar` to minimize dependencies.
Also, the Python2 bindings provided by the `zbar` package should be splitted as well or removed.